src/appsignal/cli/diagnose.py

```diff
@@ -373,7 +373,10 @@ def _file_is_writable(self, path: str) -> bool:
     def _os_distribution(self) -> str:
         try:
             return platform.freedesktop_os_release()["NAME"]
-        except OSError:
+        # The platform.freedesktop_os_release is not available in all Python
+        # versions that we support. That's why we catch AttributeError along with
+        # OSError which is thrown when the host OS doesn't comply with freedesktop.
+        except (OSError, AttributeError):
```

Comment:
It reads the same file, but it would be nice to include the whole function return value and make the formatting the same.

```
$ docker run --rm -it python
>>> import platform
>>> platform.freedesktop_os_release()
{'NAME': 'Debian GNU/Linux', 'ID': 'debian', 'PRETTY_NAME': 'Debian GNU/Linux 11 (bullseye)', 'VERSION_ID': '11', 'VERSION': '11 (bullseye)', 'VERSION_CODENAME': 'bullseye', 'HOME_URL': 'https://www.debian.org/', 'SUPPORT_URL': 'https://www.debian.org/support', 'BUG_REPORT_URL': 'https://bugs.debian.org/'}

$ docker run --rm -it python bash
root@809d031cbba5:/# cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```
The second example being how we send it for other integrations.